File: python/1066.py
oddCount = 0
evenCount = 0
posCount = 0
negCount = 0

for i in range(5):
    num = (int(input()))
    if num % 2 == 0:
        evenCount += 1
    else:
        oddCount += 1
    if num > 0:
        posCount += 1
    elif num < 0:
        negCount += 1
        
print(evenCount, 'valor(es) par(es)')
print(oddCount, 'valor(es) impar(es)')
print(posCount, 'valor(es) positivo(s)')
print(negCount, 'valor(es) negativo(s)')

File: lua/1066.lua
local evenCount = 0
local oddCount = 0
local positiveCount = 0
local negativeCount = 0

for i = 1, 5, 1 do
    local n = io.read("n")
    if n % 2 == 0 then
        evenCount = evenCount + 1
    else
        oddCount = oddCount + 1
    end

    if n > 0 then
        positiveCount = positiveCount + 1
    elseif n < 0 then
        negativeCount = negativeCount + 1
    end
end

print(evenCount.." valor(es) par(es)")
print(oddCount.." valor(es) impar(es)")
print(positiveCount.." valor(es) positivo(s)")
print(negativeCount.." valor(es) negativo(s)")